src/pages/NewWallet/CreateWalletPage.tsx

```diff
@@ -35,23 +35,27 @@ import PanelTitle from '@/components/PageComponents/PanelTitle'
 import Paragraph from '@/components/Paragraph'
 import { useStepsContext } from '@/contexts/steps'
 import { useWalletContext } from '@/contexts/wallet'
+import { useAppSelector } from '@/hooks/redux'
 import { isWalletNameValid } from '@/utils/form-validation'
 
 const CreateWalletPage = ({ isRestoring = false }: { isRestoring?: boolean }) => {
   const { t } = useTranslation()
   const { setWalletName, setPassword, walletName: existingWalletName, password: existingPassword } = useWalletContext()
   const { onButtonBack, onButtonNext } = useStepsContext()
+  const devMode = useAppSelector((s) => s.app.devMode)
 
   const [walletName, setWalletNameState] = useState(existingWalletName)
   const [walletNameError, setWalletNameError] = useState('')
   const [password, setPasswordState] = useState(existingPassword)
   const [passwordError, setPasswordError] = useState('')
   const [passwordCheck, setPasswordCheck] = useState(existingPassword)
 
+  const isDevEnvAndDevModeEnabled = import.meta.env.DEV && devMode
```

Comment:
We may use this somewhere else! Let's put this check in `utils/dev.ts`? WDYT?

(Wanted to put this in `misc` and this made me think: we should get rid of this `misc.ts` file 😆 It's a mess)

Reply:
I think I'd rather have this as a redux selector!

```ts
const devMode = useAppSelect(selectDevMode())

// In appSlice.ts
const selectDevMode = createSelector((s) => s.devMode, (devMode) => devMode && import.meta.env.DEV)
```

Reply:
Sounds good! `selectDevModeStatus` ? :)

Reply:
haha sure!